inc/class-nusa-theme-setup.php

```diff
@@ -183,6 +184,38 @@ public function add_theme_color() {
 		echo '<meta name="theme-color" content="#000" />' . "\n";
 	}
 
+	/**
+	 * Preload the URLs for our custom fonts
+	 *
+	 * @return void
+	 */
+	public function preload_fonts() {
+		?>
+		<link rel="preload" as="font" crossorigin="anonymous" href="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/fonts/fontello/fontello.svg?36463184">
```

Comment:
Th order in which these are placed is actually significant. The `woff2` files should be before the regular `woff`.
In other words, they should be in the order that the `_font-faces.scss` partial defines them.